### PR TITLE
Fe refactor

### PR DESCRIPTION
**Pull request by joel-oe-lacey**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This is a small refactor PR allowing for display of tags for debugging and easy navigation and access.

## Related Tickets & Documents
#27 

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help
- [X] no, to be added in test refactor

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by mintona — APPROVED** [review]
Thank you for adding this!

File: app/javascript/readingList/collectionForm.jsx [hunk 1]
```diff
@@ -7,10 +7,9 @@ export class CollectionForm extends Component {
   constructor({ tagList }) {
     super({ tagList });
     this.state = {
-      // eslint-disable-next-line react/no-unused-state
       title: '',
       tagList,
-      // auth: false
+      error: '',
```
Comment (philjdelong) [inline app/javascript/readingList/collectionForm.jsx:12]: Nice

File: app/javascript/readingList/collectionForm.jsx [hunk 2]
```diff
@@ -42,12 +37,12 @@ export class CollectionForm extends Component {
         window.location.href = '/readinglist';
       });
     } else {
-      console.log('Please fill out fields');
+      this.setState({ error: 'Please fill out fields' });
```
Comment (philjdelong) [inline app/javascript/readingList/collectionForm.jsx:40]: Interesting

File: app/javascript/readingList/collectionForm.jsx [hunk 3]
```diff
@@ -68,7 +63,7 @@ export class CollectionForm extends Component {
             autoComplete="off"
             classPrefix="articleform"
           />
-          {/* <h4>{`${this.state.error}`}</h4> */}
+          <h4>{`${error}`}</h4>
```
Comment (philjdelong) [inline app/javascript/readingList/collectionForm.jsx:66]: Makes sense

**Pull request merged**